src/patternfly/components/Divider/divider.scss

```diff
@@ -5,4 +5,5 @@
   height: var(--pf-c-divider--Height);
   background-color: var(--pf-c-divider--BackgroundColor);
   border: 0; // removes the default border styling on an hr
+  flex-shrink: 0;
```

Comment:
If you're not using the divider component now, do you need this?

Reply:
I think we should keep it. It keeps the element from shrinking when used as a flex child between sections since we use `height` (and presumably `width` in a vertical divider) instead of `border` or something like that. [Material's divider](https://material-ui.com/components/dividers/) also sets it.